src/directives/formly-form.test.js

```diff
@@ -280,6 +280,48 @@ describe('formly-form', () => {
       })
     })
 
+    it(`should be possible to use a wrapper & templateOptions in a fieldGroup`, () => {
+      formlyConfig.setWrapper({
+        name: 'panel',
+        template: `<div class="panel">
+            <div class="heading">
+              Panel Title: {{options.templateOptions.title}}
```

Comment:
Ah, yes, I suppose that we could set up the `to` variable in that if statement as sort of a one-off. Those other defaults probably wouldn't make sense. Do you fancy submitting a PR? Please see the CONTRIBUTING.md to get started :-)